build.gradle

```diff
@@ -22,7 +22,18 @@ task clean() {
 
   doLast {
     exec {
-      executable "./clean.sh"
+      executable "./tools/clean.sh"
+    }
+  }
+}
+
+task ssync() {
```

Comment:
It isn't immediately obvious what the first 's' is for, I would guess submodules? Since build and clean also apply to submodules maybe this prefix isn't necessary